__mocks__/jestAPICredentials.js

```diff
@@ -0,0 +1,7 @@
+export const jestAPICredentials = {
+  BASE_COMMENTS_API:
```

Comment:
Kindly explain to me why you are not using the .env file to extract the API credentials.

Reply:
Since the jest testing framework did not allow me to use process.env, I created a separate file for storing the credentials inside a js file so that I can extract them and hide them inside .gitignore.